week-6/Homework/mandatory/2-quotegenerator/quotes.js

```diff
@@ -16,10 +16,25 @@
 // pickFromArray(coloursArray)  //maybe returns "#F38630"
 //
 // You DO NOT need to understand how this function works.
+
 function pickFromArray(choices) {
   return choices[Math.floor(Math.random() * choices.length)];
 }
 
+function createQuote() {
+  const quote = document.getElementById("quote")
+  const author = document.getElementById("author")
+  quote.innerHTML = `<span>❝</span>${pickFromArray(quotes).quote}`
+  author.innerText = pickFromArray(quotes).author
```

Comment:
storying the result of pickFromArray(quotes) will be a better option, as it will apply both KISS and DRY